### PR TITLE
Support min max decimals

### PR DESCRIPTION
**Pull request by gish**

### CODE REVIEW

File: src/components/number/number.jsx [hunk 1]
```diff
@@ -61,16 +72,17 @@ export default function Number({
     whiteSpace: 'nowrap',
   }, style);
 
-  const decimals = getDecimals(value, valueDecimals, ticks);
+  const minimumFractionDigits = getFractionDigits(getTickDecimals(value, ticks), valueMinDecimals, valueDecimals);
```
Comment (bstream) [inline src/components/number/number.jsx]: You could move out getTickDecimals so it is only called once.

``` javascript
const tickDecimals = getTickDecimals(value, ticks);
const minimumFractionDigits = getFractionDigits(tickDecimals, valueMinDecimals, valueDecimals);
const maximumFractionDigits = getFractionDigits(tickDecimals, valueMaxDecimals, valueDecimals);
```

File: styleguide/number.section.md [hunk 2]
```diff
@@ -7,8 +7,10 @@ This means that all components in this section can also take these additional pr
 | Name            | Type           | Description |
 | :-------------- | :------------- | :---------- |
 | className       | string         | The class(es) that should apply to the whole component |
-| decimals        | number         | The number of decimals you want to display. For finer control, see `ticks` |
 | valueClass      | string         | The class(es) that should apply to the value part of the component |
+| valueDecimals   | number         | The number of decimals you want to display. For finer control, see `valueMaxDecimals`, `valueMinDecimals` and `ticks`. |
```
Comment (bstream) [inline styleguide/number.section.md:11]: Nice catch 👍

File: src/components/number/number.jsx [hunk 3]
```diff
@@ -23,16 +24,24 @@ function renderAddon(addon, addonClasses, addonSeparator, addonStyle, position)
...
+    return undefined;
   }
 
   const tick = ticks.find(t => value >= t.from_price && value < (t.to_price + t.tick));
 
-  return tick && typeof tick.decimals !== 'undefined' ? tick.decimals : decimals;
+  return tick ? tick.decimals : undefined;
+}
+
+function getFractionDigits(...args) {
```
Comment (bstream) [inline src/components/number/number.jsx:38]: What happens if no argument is finite?

(Eg. what happens when we set maximumFractionDigits={undefined} for the FormattedNumber?)

Comment (gish) [reply src/components/number/number.jsx:38]: Should fall back to default value 2.

File: src/components/number/number.jsx (hunk 1 above)
Comment (gish) [reply src/components/number/number.jsx]: Good point. Fixed in 7ba2cd2.

File: styleguide/number.section.md (hunk 2 above)
Comment (gish) [reply styleguide/number.section.md:11]: 😊

**Review by bstream — APPROVED** [review]

**Pull request merged**